AirCasting/Dashboard/ReorderingDashboardViewModel.swift

```diff
@@ -0,0 +1,24 @@
+// Created by Lunar on 28/01/2022.
+//
+
+import Foundation
+
+class ReorderingDashboardViewModel: ObservableObject {
+    @Published var sessions: [SessionEntity]
```

Comment:
On one hand I would really like not to see `SessionEntity` being exposed in public interfaces of viewmodels and generally being used outside of deep undersurface database layers anymore

On the other I know you based this work off of our standard dashboard and cardview and it would be super-hard to refactor this, so as sad as I am seeing such code here - let's keep it...